frontend/src/components/Section.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
Moved this component here from Settings>Account so that it may be used in Settings>Profile too